zerver/models.py

```diff
@@ -1147,7 +1147,61 @@ def __str__(self) -> str:
         return f"<Recipient: {display_recipient} ({self.type_id}, {self.type})>"
 
 
-class UserProfile(AbstractBaseUser, PermissionsMixin):
+class UserBaseSettings(models.Model):
```

Comment:
What's the thinking behind not moving "notification settings"?  I suppose we can always move those later and this helps manage complexity, but it'd be good to say that explicitly if that's the intent.